modules/sdk-core/src/bitgo/utils/opengpgUtils.ts

```diff
@@ -309,75 +309,6 @@ export async function encryptText(text: string, key: Key): Promise<string> {
   });
 }
 
-/**
- * Encrypts and detach signs a string
- * @param text string to encrypt and sign
- * @param publicArmor public key to encrypt with
- * @param privateArmor private key to sign with
- */
-export async function encryptAndDetachSignText(
```

Comment:
i recently added these openpgp utility functions but decided to move them to where they're actually needed plus change them to encrypt + decrypt and sign + verify binary data to be compatible with hsm.